x/wasm/keeper/querier_test.go

```diff
@@ -798,31 +798,51 @@ func TestQueryPinnedCodes(t *testing.T) {
 }
 
 func TestQueryParams(t *testing.T) {
-	ctx, keepers := CreateTestInput(t, false, AvailableCapabilities)
-	keeper := keepers.WasmKeeper
-
-	q := Querier(keeper)
```

Comment:
I think it is better to setting the dummy environment here than in loop.

Reply:
I fixed it.